### PR TITLE
DoD static exports superscript fix

### PR DESCRIPTION
**Pull request by ikesau**
Thanks @marcelgerber for ID-ing this one. Constructor names were getting minified, so string matching was breaking. Switched to the better technique and validated on [staging](https://ptolemy-owid.netlify.app/grapher/annual-inflation-of-consumer-prices?country=Landlocked+developing+countries+%28LLDCs%29~Least+developed+countries+%28LDCs%29~Small+Island+Developing+States+%28SIDS%29) 👍

### CODE REVIEW

File: grapher/text/MarkdownTextWrap.tsx [hunk 1]
```diff
@@ -310,17 +310,17 @@ export class IRDetailOnDemand extends IRElement {
...
-        "IRLink",
-        "IRDetailOnDemand",
-    ].includes(token.constructor.name)
+    return (
+        token instanceof IRDetailOnDemand ||
+        token instanceof IRBold ||
+        token instanceof IRSpan ||
+        token instanceof IRItalic ||
+        token instanceof IRLink
+    )
```
Comment (danielgavrilov) [inline grapher/text/MarkdownTextWrap.tsx:319]: Could this just be:

```suggestion
    return token instanceof IRElement
```

Comment (ikesau) [reply grapher/text/MarkdownTextWrap.tsx]: 🙊

**Comment by danielgavrilov** [comment]
Could they be `instanceof` expressions instead of function calls? We already have these:

https://github.com/owid/owid-grapher/blob/eebacbda402a27c5ee85b8e260b2cb7eb50dffcd/grapher/text/MarkdownTextWrap.tsx#L333

https://github.com/owid/owid-grapher/blob/eebacbda402a27c5ee85b8e260b2cb7eb50dffcd/grapher/text/MarkdownTextWrap.tsx#L364

https://github.com/owid/owid-grapher/blob/eebacbda402a27c5ee85b8e260b2cb7eb50dffcd/grapher/text/MarkdownTextWrap.tsx#L340

https://github.com/owid/owid-grapher/blob/eebacbda402a27c5ee85b8e260b2cb7eb50dffcd/grapher/text/MarkdownTextWrap.tsx#L377

**Review by marcelgerber — APPROVED** [review]

**Comment by marcelgerber** [comment]
This was not introduced in this PR, but still flagging it:
The [generated SVG](https://ptolemy-owid.netlify.app/grapher/annual-inflation-of-consumer-prices) contains a [U+2028 line separator](https://unicode-table.com/en/2028/) right after the superscript, which shows up for me as a weird sign:  
Maybe this is a Windows thing, but still something we should address and maybe change the sign used there.
This is what it looks like on Windows:
image

**Pull request merged**